server/views/account/show.jade

```diff
@@ -56,6 +56,24 @@ block content
                     if isBackEndCert
                         .button-spacer
                         a.btn.btn-primary.btn-block(href='/' + username + '/back-end-certification') View My Back End Development Certification
+                    if isFullStackCert
```

Comment:
I forgot to mention that we do not need this particular cert view button here, can you please remove this?

i.e delete lines 59-61?

Reply:
sure, will do.